lib/rules/await-fire-event.ts

```diff
@@ -19,7 +19,12 @@ export default createTestingLibraryRule<Options, MessageIds>({
     docs: {
       description: 'Enforce promises from fire event methods to be handled',
       category: 'Best Practices',
-      recommended: false,
+      recommendedConfig: {
+        dom: false,
+        angular: false,
+        react: false,
```

Comment:
I was thinking to have properties within this object optional so this one could be expressed as just:
```javascript
recommendedConfig: {
  vue: 'error'
}
```
skipping all false ones. But I actually prefer having to specify all of them, it's more explicit 👍

Reply:
I like the explicitness too, so that's why did it this way